include/main_functions_save.h

```diff
@@ -29,9 +29,10 @@ void save_corrections(char* filename, char** lines);
  *      - filename: string indicating name of file (saving destination)
  *      - lines: array of strings to be printed in file
  *      - quit: pointer to boolean (TRUE means continue in main.c loop, FALSE means exit program)
+ *      - color: pointer to boolean (TRUE means print in colors, FALSE means print without colors)
  *
  * returns: void
  */
-void save_page(char* filename, char** lines, bool* quit);
+void save_page(char *filename, char **lines, bool *quit, bool* color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!